config/_default/menus.toml

```diff
@@ -20,71 +20,84 @@
     url = "/docs/reference/cli/pulumi_destroy/"
     weight = 3
 
+[[reference]]
+    name = "History"
+    parent = "cli"
+    url = "/docs/reference/cli/pulumi_history/"
+    weight = 4
+
 [[reference]]
     name = "Login"
     parent = "cli"
     url = "/docs/reference/cli/pulumi_login/"
-    weight = 4
+    weight = 5
 
 [[reference]]
     name = "Logout"
     parent = "cli"
     url = "/docs/reference/cli/pulumi_logout/"
-    weight = 5
+    weight = 6
 
 [[reference]]
     name = "Logs"
     parent = "cli"
     url = "/docs/reference/cli/pulumi_logs/"
-    weight = 5
+    weight = 7
 
 [[reference]]
     name = "New"
     parent = "cli"
     url = "/docs/reference/cli/pulumi_new/"
-    weight = 6
+    weight = 8
 
 [[reference]]
     name = "Plugin"
     parent = "cli"
     url = "/docs/reference/cli/pulumi_plugin/"
-    weight = 7
+    weight = 9
 
 [[reference]]
     name = "Preview"
     parent = "cli"
     url = "/docs/reference/cli/pulumi_preview/"
-    weight = 8
+    weight = 10
 
 [[reference]]
     name = "Refresh"
     parent = "cli"
     url = "/docs/reference/cli/pulumi_refresh/"
-    weight = 9
+    weight = 11
 
 [[reference]]
     name = "Stack"
     parent = "cli"
     url = "/docs/reference/cli/pulumi_stack/"
-    weight = 10
+    weight = 12
 
 [[reference]]
     name = "State"
     parent = "cli"
     url = "/docs/reference/cli/pulumi_state/"
-    weight = 11
+    weight = 13
 
 [[reference]]
     name = "Update"
```

Comment:
While you're making changes here, mind changing this from "Update" => "Up" (or "up" if we're making them all lowercase). The command used to be `update` (`up` was an alias), but we changed the command to be `up` (with `update` as the alias).